ui/apps/platform/src/services/AlertsService.ts

```diff
@@ -94,13 +95,13 @@ export function fetchSummaryAlertCounts(
  * Fetch a page of list alert objects.
  */
 export function fetchAlerts(
-    searchOptions: RestSearchOption[],
+    searchOptions: SearchFilter,
```

Comment:
1. Does it help to rename `searchOptions` as `searchFilter` because the argument changes:
* from options array of objects
* to filter object
2. Also rename `options` as `searchFilter` in `fetchAlertCount` function below?